src/Vizkit3dWorld.cpp

```diff
@@ -36,9 +39,8 @@ Vizkit3dWorld::Vizkit3dWorld(std::string path,
 {
     loadGazeboModelPaths(modelPaths);
 
-    int argc = 1;
-    char const*argv[] = { "vizkit3d_world" };
-    app = new QApplication(argc, const_cast<char**>(argv));
+    if (!qApp)
+        app = new QApplication(argc, const_cast<char**>(argv));
```

Comment:
Also, avoid const_cast, specially when you don't need it

Reply:
Ok. Done. Thanks.